symfony/framework-bundle/5.1/src/Kernel.php

```diff
@@ -21,7 +21,10 @@ protected function configureContainer(ContainerConfigurator $container): void
             $container->import('../config/{services}_'.$this->environment.'.yaml');
         } else {
             $path = \dirname(__DIR__).'/config/services.php';
-            (require $path)($container->withPath($path), $this);
+
+            if (is_file($path)) {
```

Comment:
can youmerge this if with the previous else please?

Reply:
also we should be consistent with using is_file / file_exists (I'm fine using is_file above also)